tests/suites/test_suite_aes.xex.data

```diff
@@ -0,0 +1,3599 @@
+AES-128-XEX Encrypt NIST XTSTestVectors #1
```

Comment:
Which NIST document are these defined? (I couldn't find them in NIST SP 800-38E)